Mathlib/CategoryTheory/MorphismProperty/Limits.lean

```diff
@@ -254,15 +276,83 @@ lemma IsStableUnderProductsOfShape.mk (J : Type*)
   rintro ⟨j⟩
   simp [φ]
 
+/-- another version of `IsStableUnderProductsOfShape.mk`. -/
+lemma IsStableUnderProductsOfShape.mk' (J : Type*) [W.RespectsIso]
```

Comment:
This should *replace* `mk` as it is slightly more general.

Reply:
Gotcha! :)